src/Adapters/ADXAdapter.ts

```diff
@@ -85,7 +85,7 @@ export default class ADXAdapter implements IADXAdapter {
                 const resultTimeSeriesData: Array<ADXTimeSeries> = []; // considering there is going to be multiple series to fetch data for
 
                 if (adxDataHistoryResults.data) {
```

Comment:
Shouldn't this change to **adxDataHistoryResults.data && adxDataHistoryResults.data.data**?

Reply:
yes, I updated with additional checks on data.data